lib/service/service.go

```diff
@@ -1429,6 +1429,12 @@ func (process *TeleportProcess) initDiagnosticService() error {
 		trace.Component: teleport.Component(teleport.ComponentDiagnostic, process.id),
 	})
 
+	// Monitor disk space in Teleport data directory and emit to Prometheus.
+	process.RegisterFunc("disk.monitor", func() error {
+		go startDiskMonitor(process)
```

Comment:
there is no need to run this is as a `go` gourotines, as `RegisterFunc` does that for you